mod-source-record-manager-server/src/main/java/org/folio/dao/util/JobExecutionDBConstants.java

```diff
@@ -28,6 +28,7 @@ public final class JobExecutionDBConstants {
   public static final String TOTAL_FIELD = "total";
 
   public static final String GET_BY_ID_SQL = "SELECT * FROM %s WHERE id = $1";
+  public static final String DELETE_BY_IDS_SQL = "DELETE FROM %s.%s WHERE id IN ( '%s' )";
```

Comment:
Hi @SatishkumarPurswani Is there any possibility to use Tuples instead of formatting String?

Reply:
Hi @AivarIUsupov - I tried using tuple but it was not working with IN query, instead it was iterating over multiple tuples which means the number of queries will be equal to the number of JobExecution Ids that we have requested to delete.
Hence, had to use this approach.
Feel free to suggest a better approach, if any.